GREEDY/View/MainScreen.cs

```diff
@@ -20,20 +23,33 @@ public MainScreen(IReceiptService receiptService, IItemService itemService)
             _photoImageGetter = new PhotoImageGetter();
             _fileImageGetter = new FileImageGetter();
             InitializeComponent();
+            _imageFormatService = new ImageFormating();
         }
 
         private void InserFile_Button_Click(object sender, EventArgs e)
         {
             Application.UseWaitCursor = true;
             InserFile_Button.Enabled = false;
             var image = _fileImageGetter.GetImage();
-            var processedReceipt = _receiptService.ProcessReceiptImage(image);
-            if (processedReceipt != null)
+
+            //PointSelector pointSelector = new PointSelector(image);
+            //pointSelector.Show();
```

Comment:
These comments can be deleted now since we wont ask user to select the points of the receipt.

Reply:
thanks